src/components/registerV2/confirmPassphrase.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import fillWordsList from 'bitcore-mnemonic/lib/words/english';
+import { translate } from 'react-i18next';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { FontIcon } from '../fontIcon';
+import { PrimaryButtonV2, SecondaryButtonV2 } from '../toolbox/buttons/button';
+import registerStyles from './registerV2.css';
+import styles from './confirmPassphrase.css';
+import lock from '../../assets/images/icons-v2/lock.svg';
+import Options from './confirmPassphraseOptions';
+
+class ConfirmPassphrase extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      numberOfOptions: 3,
+      numberOfWords: 2,
+      words: [2, 9],
+      tries: 0,
+      maxTries: 3,
+      answers: [],
+      options: [],
+      hasErrors: false,
+      isCorrect: false,
+      outOfTries: false,
+    };
+
+    this.handleSelect = this.handleSelect.bind(this);
+    this.handleConfirm = this.handleConfirm.bind(this);
+    this.verifyChoices = this.verifyChoices.bind(this);
```

Comment:
Most of the current codebase uses .bind(this) before the method is used in JSX. Is there any advantage of this way?

Reply:
This is the way that is recommended right know at the React documentation, and also doing the binding in the render method, may cause some perform issues, almost always unnoticeable.
Because when binding the function it generates a new reference for the function, so if we bind it in the constructor, it keeps the same reference, but binding on render, when React compares the virtual DOM for changes, it will consider the function as being a new one all the time.